#! /bin/sh /usr/share/dpatch/dpatch-run
## 51_overwrite_output.dpatch by Petter Reinholdtsen

Include more information when overwriting the provide LSB defaults.

@DPATCH@
diff -u insserv-1.11.0/insserv.c insserv-1.11.0/insserv.c
--- insserv-1.11.0/insserv.c
+++ insserv-1.11.0/insserv.c
@@ -2460,8 +2460,9 @@
 			     */
 			    if (!defaults && (deflvls != service->lvls)) {
 				if (!del && chkfor(d->d_name, argv, argc) && !(argr[curr_argc]))
-				    warn("Warning, current runlevel(s) of script `%s' overwrites defaults.\n",
-					 d->d_name);
+				    warn("Warning, current start runlevel(s) of script `%s' (%s) overwrites defaults (%s).\n",
+					 d->d_name, lvl2str(service->lvls),
+					 lvl2str(deflvls));
 			    }
 			} else
 			    /*
@@ -2500,8 +2509,9 @@
 			     */
 			    if (!defaults && (deflvlk != service->lvlk)) {
 				if (!del && chkfor(d->d_name, argv, argc) && !(argr[curr_argc]))
-				    warn("Warning, current runlevel(s) of script `%s' overwrites defaults.\n",
-					 d->d_name);
+				    warn("Warning, current stop runlevel(s) of script `%s' (%s) overwrites defaults (%s).\n",
+					 d->d_name, lvl2str(service->lvlk),
+					 lvl2str(deflvlk));
 			    }
 			} else
 			    /*
